src/components/Badge/index.tsx

```diff
@@ -131,7 +131,7 @@ const Badge: FunctionComponent<BadgeProps> = ({
     [prominence, theme],
   )
 
-  const sizeValue = SIZES[size]
+  const sizeValue = SIZES[size] || SIZES.medium
```

Comment:
```suggestion
  const sizeValue = SIZES?.[size] ?? SIZES.medium
```

?

Reply:
Can we also ad a test ?

Reply:
more like `SIZES[size] ?? SIZES.medium` as `SIZES` always exists, I will add testing :)

Reply:
more like `SIZES[size] ?? SIZES.medium` as `SIZES` always exists, I will add testing :)